modules/http/http-worker.c

```diff
@@ -509,6 +525,7 @@ _custom_map_http_result(HTTPDestinationWorker *self, const gchar *url, HttpRespo
                  evt_tag_str("action", "disconnect"),
                  evt_tag_str("url", url),
                  evt_tag_int("status_code", http_code),
+                 evt_tag_mem("response", self->response_buffer->str, self->response_buffer->len),
```

Comment:
A few of these messages are on the "notice" level, not on debug. 

Nevertheless, they seem useful and hopefully they are triggered rarely under normal circumstances.